eng/pipelines/pr.yml

```diff
@@ -10,3 +10,5 @@ pr:
 
```

Comment:
Could you as a follow-up rename pr.yml to pullrequest.yml and update the pipeline name similarly to align with the other languages.

Reply:
created issue #2084